src/page/template/content/conversation.htm

```diff
@@ -1,12 +1,11 @@
 <div id="conversation" class="conversation" data-bind="drop_file: $root.inputBar.onDropFiles, css: { loading: !$root.messageList.conversationLoaded() }">
   <!-- ko if: $root.conversationRepository.active_conversation() -->
-    <div class="conversation-loading" >
-      <div class="icon-spinner spin text-theme"></div>
-    </div>
-    
     #include('content/conversation/title-bar.htm')
     #include('content/conversation/message-list.htm')
     #include('content/conversation/input-bar.htm')
+    <div class="conversation-loading" >
```

Comment:
This will fix a bug where we would trigger the callbacks supposed to be triggered when a message is viewed by the user (updating conversation last_read_timestamp, sending read receipts...) while the conversation was loading (and thus the messages not visible to the user).